CIP-data-compression/README.md

```diff
@@ -0,0 +1,201 @@
+---
+CIP: ?
+Title: Data compression for higher throughput and lower latency
```

Comment:
```suggestion
Title: Block Data Compression
```

Reply:
continued as of https://github.com/cardano-foundation/CIPs/pull/993#issuecomment-2684720453 & will remain highlighted for discussn @ next CIP meeting.

Reply:
resolving, since this was thankfully fixed in one of the earlier revisions 😅